README.md

```diff
@@ -1,6 +1,44 @@
 # ViperDefender
+ViperDefender is a python based client that can be used to detect and remove malware from a system. It uses the [VirusTotal]() API to scan files and a cloud sanbox that test every files downloaded on the Computer. It also has a public API that can be used to scan files.
 
-The Mastercamp project
+## Table of Contents
+- [ViperDefender](#viperdefender)
+  - [Table of Contents](#table-of-contents)
+  - [Authors](#authors)
+  - [Requirements](#requirements)
+  - [Installation](#installation)
+  - [Usage](#usage)
+  - [API](#api)
 
+## Authors
+- [Matthieu Roquejoffre](https://github.com/Mattherix)
+- [Pierre-Marie Herrburger--Pietri](https://github.com/Unaxe)
+- [Mathieu Lebrun](https://github.com/OursCarrock)
+- [Evan Zanzuchi](https://github.com/NeverlandOZ)
+- [Victor Boulet](https://github.com/Viki922)
+- [Theodore Massima](https://github.com/watchiMass)
```

Comment:
We could also use [contrib.rock](https://contrib.rocks/preview?repo=Mattherix%2FViperDefender). I think it looks nicer.
<a href="https://github.com/Mattherix/ViperDefender/graphs/contributors">
  <img src="https://contrib.rocks/image?repo=Mattherix/ViperDefender" />
</a>

```suggestion
<a href="https://github.com/Mattherix/ViperDefender/graphs/contributors">
  <img src="https://contrib.rocks/image?repo=Mattherix/ViperDefender" />
</a>
```